docs/articles/testkube-dashboard-execution-details.md

```diff
@@ -6,7 +6,15 @@ When selecting a specific Workflow Execution the modal that opens has the follow
 
 This tab shows the log output for each executed step - [Read More](/articles/logs-and-artifacts#log-collection).
 
-![Log Output](../img/workflows-log-output-2.1.png)
+![Log Output](../img/workflows-log-output-092024.png)
```

Comment:
we should add this and the below section on comparison to https://docs.testkube.io/articles/logs-and-artifacts also

Reply:
Done